src/cmap/connection.ts

```diff
@@ -88,6 +89,11 @@ export interface CommandOptions extends BSONSerializeOptions {
   writeConcern?: WriteConcern;
 
   directConnection?: boolean;
+
+  /** @internal */
+  timeout?: Timeout;
+  /** @internal */
+  serverSelectionTimeout?: Timeout;
```

Comment:
Added to facilitate the separation of the overall operation timeout and the server selection timeout requirements